src/golang/lib/workflow/artifact/artifact.go

```diff
@@ -126,11 +126,11 @@ func (a *ArtifactImpl) Name() string {
 }
 
 func (a *ArtifactImpl) Computed(ctx context.Context) bool {
-	// An artifact is only considered computed if its results have been written.
+	// An artifact is only considered computed if its contents have been written.
 	res := utils.ObjectExistsInStorage(
 		ctx,
 		a.storageConfig,
-		a.execPaths.ArtifactMetadataPath,
```

Comment:
are we still using `metadataPath`?

Reply:
Yeah, it just won't be used to checking if artifact was "computed". The metadata path and output path used to be written in tandem, but in certain edge cases (eg. failed parameter operator), it's possible that the contents are written but no the metadata path.